code/modules/ruins/spaceruin_code/clownstation.dm

```diff
@@ -0,0 +1,56 @@
+///////////	clownstation items
+
+/obj/item/paper/fluff/ruins/clownstation/dream
+	info = "Last night the Honkmother spoke to me in a dream. Though I've dedicated my life to religious study, I was not prepared for such direct contact! She gave me a grave warning. She said there was danger in the monastery, and that I may not survive. She also consoled me, showing me a vision of Honkven in all of its glory. It was more beautiful than I ever could have imagined."
+
+/obj/item/paper/fluff/ruins/clownstation/mimenote
+	info = "silentio omnes consumentur"
+
+/obj/item/paper/fluff/ruins/clownstation/entry
+	name = "visitor notice"
+	info = "While the Honk Salvation Monastery welcomes pilgrims and visitors alike, please be mindful of the monks who live here and their needs. Additionally, please wear clown religious garb for the duration of your time visiting. If you borrow one of our masks, make sure to return it when you leave."
+
+/obj/item/paper/fluff/ruins/clownstation/honkfather
+	name = "visitor notice"
+	info = "Many visitors ask why we choose to display a statue of the Honkfather. The Honkfather is a deeply controversial figure, oftern thought to represent the dangers of pranking in bad fun and the way pranking can turn into crime. We choose to honor the Honkfather, because although he is imperfect, he is a member of the divine harlequin and ultimately a holy force for good. Let us not forget his actions in the book of Bananacus, where he cured the tribe of Pierrots of their sadness, or his prophesided role in The Great World Peeling, where it is said he will fight by the Honkmother's side with his duel tommy guns. An additional note about our statue- the gun used is a replica and cannot fire."

```

Comment:
*dual tommy guns

also I thought that clowns were clumsy and can't use guns very well